### PR TITLE
Update kubernetes documentation for users

### PR DESCRIPTION
**Pull request by emilywoods**

### CODE REVIEW

File: docs/scaling/kubernetes.txt [hunk 1]
```diff
@@ -135,6 +135,13 @@ Find out the public IP and the services that have been created with
 
     kubectl get services
 
+If it is acceptable that this service is exposed externally, then specifying the 
+service type as `LoadBalancer` is sufficient. An external load balancer then 
+routes to `NodePort` and `ClusterIP` services. However, if it is not acceptable
+to expose the database on an external (public) IP, we recommend specifying the
+type as NodePort. These means that the service is exposed on each Node's IP as
```
Comment (turbo-ele) [inline docs/scaling/kubernetes.txt]: This instead of these?

File: docs/scaling/kubernetes.txt [hunk 2]
```diff
@@ -185,17 +192,46 @@ allows for a better flexibility than passing around config files.
 Storage
 -------
 
+The above example makes use of in-memory storage. This is only suggested to 
+facilitate testing out of CrateDB with Kubernetes. It is not advisable to use
+this form of storage for a production database because all data from this
+directory will be lost if a pod is deleted. More information on this can be
+found `here <https://kubernetes.io/docs/concepts/storage/volumes/>`, under
```
Comment (turbo-ele) [inline docs/scaling/kubernetes.txt]: you could remove the `under emptyDir` and provide the link as follows https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

File: docs/scaling/kubernetes.txt [hunk 3]
```diff
@@ -185,17 +192,46 @@ allows for a better flexibility than passing around config files.
...
+      - metadata:
+          name: crate-persistent-storage
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 10Gi
+
+Each PV has a lifecycle which is independent of the pod which makes use of it.
```
Comment (turbo-ele) [inline docs/scaling/kubernetes.txt]: would replace the second which with `that`

File: docs/scaling/kubernetes.txt [hunk 4]
```diff
@@ -185,17 +192,46 @@ allows for a better flexibility than passing around config files.
...
+however the performance of such volumes can be dependent on the cloud provider.
+Nevertheless, replication only makes sense if the data is on separate physical
+media in order to achieve high availability and to avoid potential bottlenecks
+when several clusters attach the same storage. More information about
+provisioning persistent volumes with a cloud provider can be found `here
+<https://github.com/kubernetes/examples/tree/master/staging/persistent-volume-provisioning>`.
+
+You can familiarize yourself with `Volumes` and `StatefulSets` `here
+<https://kubernetes.io/docs/concepts/workloads/controllers/statefulset>`_.
+Volumes can be customized as required, to achieve this, the `volume guide
```
Comment (turbo-ele) [inline docs/scaling/kubernetes.txt]: the ``kubernetes volume guide`` ...

File: docs/scaling/kubernetes.txt (hunk 2 above)
Comment (emilywoods) [reply docs/scaling/kubernetes.txt]: it doesn't actually take you to that section though...

**Comment by autophagy** [comment]
@emilywoods I think it should be re-referenced to https://crate.io/docs/crate/reference/config/index.html

**Pull request merged**